src/components/cc-button/cc-button.js

```diff
@@ -123,6 +123,10 @@ export class CcButton extends LitElement {
 
     e.stopPropagation();
 
+    if (this.disabled) {
```

Comment:
Can we add a comment here where we explain that because we use `aria-disabled` we need to take care of this.